src/containers/sidebars/tabs-sidebar/component.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from 'react';
```

Comment:
There is a mismatch in the name we give to the files: some components contain the name of the same in each file, others contain only the dimension (index, selectors, etc). I prefer the second way of calling them because I think it is easier to search and less visually polluting, but we can decide which format is more interesting so that we can always use the same way.

Reply:
For me is easier to search as you can search by the tabs-sidebar-component file directly